packages/spacecat-shared-rum-api-client/src/index.js

```diff
@@ -22,6 +23,7 @@ import highOrganicLowCtr from './functions/opportunities/high-organic-low-ctr.js
 const HANDLERS = {
   404: notfound,
   cwv,
+  formVitals,
```

Comment:
thanks 👍

Reply:
@05rahulsingh how about `form-vitals` to be in sync with other handlers?

Reply:
@rpapani done.